doc/merge_split.rst

```diff
@@ -0,0 +1,19 @@
+*Merge and Split 
+======================
+
+
+This submodule is a post processing step to address tracked cells which merge/split. 
+The first iteration of this module is to combine the cells which are merging but have received a new cell id (and are considered a new cell) once merged. 
+This submodule will label merged/split cells with a TRACK number in addition to its CELL number.
+
+Features, cells, and tracks are combined using parent/child nomenclature. 
+(quick note on terms; “feature” is a detected object at a single time step. “cell” is a series of features linked together over multiple timesteps. "track" may be an individual cell or series of cells which have merged and/or split.)
+
```

Comment:
On the one hand, while I think the documentation to the function should not include references to the MST, I think this (or a second page) would be a good place to document how the mergers/splits function actually works, below the output dataframe.